app/components/donation/donation_card_component.rb

```
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Donation::DonationCardComponent < ViewComponent::Base
+    def initialize(image_path:, link:, alt_text:, description:)
```

RuboCop (1.75.5)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)